src/mame/drivers/galaxian.cpp

```diff
@@ -5079,38 +5079,26 @@ static INPUT_PORTS_START( thepitm )
 	PORT_BIT( 0x08, IP_ACTIVE_HIGH, IPT_JOYSTICK_RIGHT ) PORT_COCKTAIL
 	PORT_BIT( 0x10, IP_ACTIVE_HIGH, IPT_JOYSTICK_UP ) PORT_COCKTAIL
 	PORT_BIT( 0x20, IP_ACTIVE_HIGH, IPT_BUTTON1 ) PORT_COCKTAIL
-	PORT_DIPNAME( 0x40, 0x00, DEF_STR( Unknown ) ) // turning both of these on boots with 9 credits?
-	PORT_DIPSETTING(    0x00, DEF_STR( Off ) )
-	PORT_DIPSETTING(    0x40, DEF_STR( On ) )
-	PORT_DIPNAME( 0x80, 0x00, DEF_STR( Unknown ) )
-	PORT_DIPSETTING(    0x00, DEF_STR( Off ) )
-	PORT_DIPSETTING(    0x80, DEF_STR( On ) )
+	PORT_DIPNAME( 0xc0, 0x00, DEF_STR( Coinage ) )
+	PORT_DIPSETTING(    0x00, "1 Credit per Player" )
+	PORT_DIPSETTING(    0x80, "2 Credits/1 Player, 3 Credits/2 Players" )
+	PORT_DIPSETTING(    0x40, "2 Credits per Player" )
```

Comment:
This is just a standard 1C/1P, 3C/2P, 2C/1P setup.  There’s no reason to use custom strings.

Reply:
It’s just showing the actual number of coins though – it’s like the later Midway games in that way.  They typically default to 2 coins per play.  You put in one coin, it shows one coin, and says “insert 1 more coin” or something.  We still call that setting 2 Coins/1 Credit whether the machine displays the raw number of coins or the number of starts you’ve paid for.

Reply:
I have implemented the change as advised.  I will make a similar change to the original thepit set for consistency later.  Thanks